assets/js/components/Table/Table.jsx

```diff
@@ -148,7 +149,13 @@ function Table({
         />
       </div>
       <div className="">
-        <div className="-mx-4 sm:-mx-8 px-4 sm:px-8 pt-4">
+        <div
+          className={
```

Comment:
you could also use `classNames`.

```js
import classNames from 'classnames';
...
<div
  className={classNames('-mx-4 sm:-mx-8 px-4 sm:px-8', {'pt-4': withPadding})}
...
```

Reply:
Woh damn you're right, totally overlooked that

Reply:
aye this is great thanks